handler.js

```diff
@@ -1,14 +1,66 @@
 'use strict';
+// import FilesReader and SkillsWriter classes from skills-kit-2.0.js library
+const { FilesReader, SkillsWriter } = require('skills-kit-library/skills-kit-2.0');
+const request = require('request-promise');
+//const BoxSDK = require('box-node-sdk');
+//const fs = require('fs');
+//const boxSdkConfig = JSON.parse(fs.readFileSync('./conf/boxSdkConfig.json', 'utf8'));
```

Comment:
leaving only because still need to test methods of creating client with auth to share a file